<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformerPage</class>
 <widget class="QDialog" name="InformerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabInformer">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSystem">
      <attribute name="title">
       <string>Системные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblSystemCreatePerson">
         <property name="text">
          <string>Автор</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="lblSystemCreatePersonValue">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSystemCreateDatetime">
         <property name="text">
          <string>Дата</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="lblSystemCreateDatetimeValue">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSystemSubject">
         <property name="text">
          <string>Тема</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QTextBrowser" name="edtSystemText">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDialogButtonBox" name="systemButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::NoButton</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblSystemText">
         <property name="text">
          <string>Текст</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="lblSystemSubjectValue">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkSystemMarkViewed">
         <property name="text">
          <string>я ознакомлен(а)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExternal">
      <attribute name="title">
       <string>Внешние</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="CTableView" name="tblExternalEvent"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkExternalMarkViewed">
         <property name="text">
          <string>закрыть уведомления</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkExternalShowEventsClosed">
         <property name="text">
          <string>показать закрытые уведомления</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabInformer</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>edtSystemText</tabstop>
  <tabstop>chkSystemMarkViewed</tabstop>
  <tabstop>systemButtonBox</tabstop>
  <tabstop>tblExternalEvent</tabstop>
  <tabstop>chkExternalMarkViewed</tabstop>
  <tabstop>chkExternalShowEventsClosed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
